### DISCUSSION TITLE
About "parse_cookie_file" function in "utils.py" file

### DISCUSSION THREAD

**Discussion by 4n1qz5skwv** [kevinzg/facebook-scraper#392] [Ideas]
Is it possible to make a _pypi python package_ for **_parse_cookie_file_** function in **_utils.py_** file. This function is super useful. Will be very helpful to use it in diffrent scraping projects. 

Thank you.

**Comment by neon-ninja** [comment]
Why not use https://docs.python.org/3/library/http.cookiejar.html?

Or just `from facebook_scraper.utils import parse_cookie_file`

**Reply by 4n1qz5skwv** [reply]
> Why not use https://docs.python.org/3/library/http.cookiejar.html?

-Well, `http.cookiejar` doesn't work as good as your function. Its also very limited with the formats.

> Or just `from facebook_scraper.utils import parse_cookie_file`

-Yes, currently using it like this. 

#
It works very well. Also i didn't find any similar package like this. So i thought maybe making it a **standalone package** was a good idea. 
This will also avoid unnecessary *requirements from installing. 

*`requirements` that needed for `facebook-scraper` but not for `parse_cookie_file`

**Reply by 4n1qz5skwv** [reply]
Also there is a chance of function name getting change or move, because its not the primary function.

**Comment by kevinzg** [comment]
This is how youtube-dl does it, might be helpful if you are creating a package.

[YoutubeDLCookieJar](https://github.com/ytdl-org/youtube-dl/blob/master/youtube_dl/utils.py#L2740)
